### PR TITLE
Hotfix/restore coverage dates

### PR DESCRIPTION
**Pull request by noahmanger**
Once the API fix for coverage dates is in (https://github.com/18F/openFEC/pull/2418) this can be merged to restore them to the frontend. I also rolled in the 1 character change from https://github.com/18F/openFEC-web-app/pull/2073.

### CODE REVIEW

File: openfecwebapp/app.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def get_absolute_url():
 
 def get_canonical_url():
     path = furl.furl(request.url).path
-    return '{}/data{}'.format(config.canonical_base, path)
+    return '{}{}'.format(config.canonical_base, path)
```
Comment (ccostino) [inline openfecwebapp/app.py]: Just confirming we don't need the `/data` here - I know we changed some things around this but wanted to be sure!

File: openfecwebapp/templates/search-results.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
     <header>
       <h1 class="heading--main">Candidate and committee profiles</h1>
     </header>
-    <form action="/" method="GET" class="slab slab--inline slab--neutral">
+    <form action="" method="GET" class="slab slab--inline slab--neutral">
```
Comment (ccostino) [inline openfecwebapp/templates/search-results.html]: Also confirming that we don't need the `/` here either.

File: openfecwebapp/app.py (hunk 1 above)
Comment (noahmanger) [inline openfecwebapp/app.py]: Yep. The `/data` is actually included in the `path` already.

File: openfecwebapp/templates/search-results.html (hunk 2 above)
Comment (noahmanger) [inline openfecwebapp/templates/search-results.html]: Nope. That was causing the form to submit to the home page and not the main data landing page.

**Pull request merged**